tools/local-test/scripts/test-by-ibcmd.sh

```
@@ -2,44 +1,107 @@
+mkdir -p $TMP_PATH
+mkdir -p $RESULT_PATH
+
+rm -rf $TMP_PATH/*
+rm -rf $RESULT_PATH/*
+rm -rf $LOGS_PATH/*
+
+cat >~/.1cv8/1C/1cv8/conf/conf.cfg <<EOL
+DisableUnsafeActionProtection=.*
+EOL
+
+declare -A SOURCES_HASH
```

Shellcheck (0.10.0)

[warning] SOURCES_HASH appears unused. Verify use (or export if used externally).

(SC2034)